ios/RNSentry.m

```diff
@@ -91,6 +86,43 @@ + (BOOL)requiresMainQueueSetup {
     resolve(@YES);
 }
 
+- (void)setEventOriginTag:(SentryEvent *)event {
+  if (event.sdk != nil) {
+    NSString *sdkName = event.sdk[@"name"];
+
+    // If the event is from react native, it gets set there and we do not handle
+    // it here.
+    if ([sdkName isEqualToString:@"sentry.cocoa"]) {
+      [self setEventEnvironmentTag:event origin:@"ios" environment:@"native"];
+    }
+  }
+}
+
+- (void)setEventEnvironmentTag:(SentryEvent *)event
+                        origin:(NSString *)origin
+                   environment:(NSString *)environment {
+  NSMutableDictionary *newTags = [NSMutableDictionary new];
+  if (nil != event.tags) {
+    [newTags addEntriesFromDictionary:event.tags];
+  }
+  [newTags setValue:origin forKey:@"event.origin"];
```

Comment:
There's no nullability annotation for `origin` and `environment`.
We could enable it on this file and fix it, but for this PR maybe good enough to check for nulls for each before trying to add the tag